views/foyer.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Welcome <%= @player_name %>!</h1>
+<h2>Choose your move:</h2>
+<form action="/choice" method="post">
+<button type="sumbit" name="choice" value="Rock">Rock</button>
+<button type="sumbit" name="choice" value="Scissors">Scissors</button>
+<button type="sumbit" name="choice" value="Paper">Paper</button>
+
```

Comment:
do you need to close this form with /form ?

(I can't add the proper syntax or it won't post the comment for some reason)